### PR TITLE
Sscs 7307 mya add better logging

### PR DESCRIPTION
**Pull request by srochani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: app/server/controllers/login.ts [hunk 1]
```diff
@@ -142,7 +142,7 @@ function getIdamCallback(
           }
         } else {
           const hearingsByName = getHearingsByName(hearings);
-
+          AppInsights.trackTrace(`[Cases count ${hearings.length}] - User logged in successfully as ${email}`);
```
Comment (jackmaloney) [inline app/server/controllers/login.ts]: What does 'Cases count {hearings length}' mean?

Comment (srochani) [reply app/server/controllers/login.ts]: Its list of cases, in MYA user either can land on only one case by providing the tya number or  on summary page without tya which shows list of cases

Comment (jackmaloney) [reply app/server/controllers/login.ts]: but why is it looking at the hearing count?

Comment (srochani) [reply app/server/controllers/login.ts]: hearing object is the case details in MYA

**Comment by srochani** [comment]
Closing and opening new PR

**Pull request closed without merging**